client/src/pages/About/About.jsx

```diff
@@ -1,12 +1,71 @@
 import React from 'react'
 import './About.css'
+import flowers3 from '../../assets/Flowers3.png'
+import flowers2 from '../../assets/Flowers2.png'
+import treesBig from '../../assets/TreesBig.png'
+import treesSmall from '../../assets/TreesSmall.png'
+import ladyLaptop from '../../assets/LadyLaptop.png'
+import manLaptop from '../../assets/ManLaptop.png'
 
 const About = () => {
     return (
         <div className="page-content">
-            <h1>About us</h1>
+            <img src={flowers3} alt = "three flowers" 
+                style = {{position: 'relative', left: 260, top: 113, height: '3.187%', width: '3.187%'}}
+            />
```

Comment:
Don't worry too much about it, but in the future, it's better practice to add classes to your html and then put your css in the `About.css` file, rather than adding styles straight to the code. It makes it much easier to reuse code.

Reply:
updated css and pushed to master!